plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/write/TestMapredParquetOutputFormat.java

```diff
@@ -40,6 +41,10 @@
 public class TestMapredParquetOutputFormat
         extends MapredParquetOutputFormat
 {
+    static {
+        replaceHadoopParquetMemoryManager();
```

Comment:
- let's add a code comemnt why it's done here
- rename this class. It's called `Test*`, but it's not a test. Should be `Testing*`